pkgs/os-specific/linux/firmware/fwupd/drop-markdown-check.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/docs/meson.build b/docs/meson.build
+index 92b9e6fc7..56bc189d9 100644
+--- a/docs/meson.build
++++ b/docs/meson.build
+@@ -1,4 +1,4 @@
+-if gidocgen_dep.found() and docs_python_deps.allowed() and gidocgen_app.found() and introspection.allowed()
++if gidocgen_dep.found() and gidocgen_app.found() and introspection.allowed()
+   toml_conf = configuration_data()
+   docgen_version = source_version
+   if git.found() and source_version != fwupd_version
+diff --git a/meson.build b/meson.build
+index 0ac74a510..a3b1ad5e8 100644
+--- a/meson.build
++++ b/meson.build
+@@ -539,18 +539,12 @@ fwupd_gir = []
+ gir_dep = dependency('gobject-introspection-1.0', required: get_option('introspection'))
+ introspection = get_option('introspection').disable_auto_if(host_machine.system() != 'linux').disable_auto_if(not gir_dep.found())
+ 
+-markdown_version = run_command(
+-  python3, '-c', 'import markdown; print(markdown.__version__)'
+-).stdout().strip()
+-docs_python_deps = get_option('docs').require(markdown_version.version_compare('>=3.2'),
```

Comment:
Maybe 

```suggestion
-markdown_version = run_command(
-  python3, '-c', 'import markdown; print(markdown.__version__)'
-).stdout().strip()
-docs_python_deps = get_option('docs').require(markdown_version.version_compare('>=3.2'),
+docs_python_deps = get_option('docs').require(true,
```

will be simpler.

Reply:
Fair, that'll be easier to rebase in the future. I'll fix it up later when I'm near a computer again.